jdbc-pool/9.0.65.wso2v1/pom.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+   ~ Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
```

Comment:
Add the correct license header

Reply:
addressed with [5a3082c](https://github.com/wso2/orbit/pull/812/commits/5a3082c550585b9966ca1b595d7c470d3bdd45ed)